src/components/typography/typography.component.tsx

```diff
@@ -170,17 +217,17 @@ const Typography = styled.span.attrs(
     line-height: ${lineHeight};
     margin: ${defaultMargin};
     padding: 0;
-    white-space: ${truncate ? "nowrap" : whiteSpace};
-    word-wrap: ${wordWrap};
-    text-overflow: ${truncate ? "ellipsis" : textOverflow};
-    ${truncate &&
+    white-space: ${truncate ? "nowrap" : (whiteSpace as string)};
```

Comment:
Instead of assertions you should assign an interface for arguments of this function in line 211

Reply:
I don't think we should assert these (and similar props below) `as string` anyway, because the props. interface allows them to be undefined, which is a valid value for the CSS properties as well (ie they can just not be set). I'm not sure how best to fix the TS errors here but it looks like for some reason TS thinks all these props are `unknown` so I'd rather figure out how to fix this issue than assert props are `strings` which might not be.

Reply:
Yeah, will take a look Robin - the linting tool was being here too so just kept it to simple assertions